spec.html

```diff
@@ -48948,11 +48918,13 @@ <h1>
   <emu-clause id="sec-relations-of-candidate-executions">
     <h1>Relations of Candidate Executions</h1>
 
-    <emu-clause id="sec-agent-order" aoid="agent-order">
-      <h1>agent-order</h1>
-      <p>For a candidate execution _execution_, _execution_.[[AgentOrder]] is a Relation on events that satisfies the following.</p>
+    <p>The following relations and mathematical functions are parameterized over a particular candidate execution and orders its events.</p>
+
+    <emu-clause id="sec-agent-order">
+      <h1>is-agent-order-before</h1>
+      <p>For a candidate execution _execution_, its <dfn>is-agent-order-before</dfn> Relation is the least Relation on events that satisfies the following.</p>
```

Comment:
Can we `<emu-xref>` these usages of "least" to the sentence in [6.2.3 The Set and Relation Specification Types](https://tc39.es/ecma262/#sec-set-and-relation-specification-type) that says

> A Relation is least with respect to some conditions when it is the smallest Relation that satisfies those conditions.

Or maybe just `<dfn>` it as "least Relation"?